test/suites/ipv6/suite_test.go

```diff
@@ -94,4 +97,18 @@ var _ = Describe("IPv6", func() {
 		})
 		Expect(internalIPv6Addrs).To(HaveLen(1))
 	})
+	It("should provision a static IPv6 prefix with node launch and set IPv6 as primary in the primary network interface", func() {
+		output, err := env.EC2API.DescribeInstances(env.Context, &ec2.DescribeInstancesInput{
+			InstanceIds: []string{coretest.Pod().Spec.NodeName},
+		})
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(output.Reservations[0].Instances[0].NetworkInterfaces[0].Ipv6Prefixes).To(HaveLen(1))
+		ipv6IsPrimary := func(ips []types.InstanceIpv6Address) bool {
+			_, yes := lo.Find(ips, func(ip types.InstanceIpv6Address) bool {
```

Comment:
```suggestion
			_, ok := lo.Find(ips, func(ip types.InstanceIpv6Address) bool {
```

nit: More standard syntax